### DISCUSSION TITLE
Add a new API for tuner "get_best_parameter"

### DISCUSSION THREAD

**Discussion by xuehui1991** [microsoft/nni#3778] [Ideas]
**What would you like to be added**:

**Why is this needed**:

**Without this feature, how does current nni work**：

**Components that may involve changes**:

**Brief description of your proposal if any**:

**Comment by scarlett2018** [comment]
Assign to @xuehui1991 for more information.

**Comment by scarlett2018** [comment]
@xuehui1991 - is the best parameter from all the completed trials? or to get a "best" parameter predication based on all the info you current have, it is a predication. if it is the later case, it seems similar to getNextParameter().

cc @chicm-ms @QuanluZhang

**Comment by savan77** [comment]
Hi @scarlett2018 and @xuehui1991 is there any updates on this? I am running hyper parameter optimization trials, but once they are completed I want to know which trial had the highest accuracy/lowest loss? 
Is there a way to get this? If you can provide some guidance around this I can also send a PR. Thanks
